packages/reporter-vue/src/commands/CommandRow.spec.ts

```diff
@@ -0,0 +1,238 @@
+import RunnablesList from '../runnables/RunnablesList.vue'
+import CommandRow from './CommandRow.vue'
+import { Runnables } from '../store/reporter-store'
+import { defineComponent, h, computed, ref } from 'vue'
+import IconEyeSlash from 'virtual:vite-icons/fa/eye-slash'
+import _, { size } from 'lodash'
+
+const passedCommand = {
```

Comment:
These seem duplicated, could we put them in a file and import them?

Reply:
Yeah. I was kinda waiting on the GQL stuff because I heard that we were going to do a better job of auto-generating good mocks. But.... that's not gonna come. So I'll invest a bit more in cleaning up these fixtures.

Reply:
I think it will, but not for a while